Resources/PO/LoginPage.robot

```diff
@@ -0,0 +1,28 @@
+*** Settings ***
+Library  SeleniumLibrary
+
+
+*** Variables ***
+${LOGIN_USERNAME_LOCATOR} =  id:user_email
```

Comment:
my advice to use single strategy of locators type. 
convert this two to xpath format